engine/src/main/java/org/terasology/math/AABB.java

```diff
@@ -46,10 +46,24 @@ private AABB(Vector3f min, Vector3f max) {
         this.max = max;
     }
 
+    /**
+     * Creates a new AABB from the given minimum and maximum points.
+     *
+     * @param min The minimum of the AABB.
+     * @param max The maximum of the AABB.
+     * @return The created AABB.
+     */
     public static AABB createMinMax(Vector3f min, Vector3f max) {
         return new AABB(new Vector3f(min), new Vector3f(max));
     }
 
+    /**
+     * Creates a new AABB with the given center and extents.
+     *
```

Comment:
Can you provide an example? Like creating an AABB around 0,0,0 with extent like 1,2,3. What would be the result?. At least that is something I would like to know when reading the documentation.
The current sentence adds no more information than the method and parameter names which already exist.